index.html

```diff
@@ -55,6 +55,157 @@ <h6>I'am a Software Developer</h6>
+                </div>
+                <div class="project-3">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-4">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-5">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-6">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+            </div>
```

Comment:
you have learned about Grid CSS so this is your opportunity to use what you've learned about Grid, so please try to use `display: grid;` in this section

Reply:
done